src/admin-guide/environments/basic.yml

```diff
@@ -0,0 +1,134 @@
+name: cylc-8-basic
+# Cylc 8 without GUI installed from Conda Repos.
+channels:
+  - defaults
+  - conda-forge
+  - kinow
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=4.5=1_gnu
+  - aiofiles=0.5.0=py37_0
+  - alembic=1.6.4=pyhd3eb1b0_0
+  - aniso8601=7.0.0=py_0
+  - ansimarkup=1.5.0=pyh44b312d_0
+  - asn1crypto=1.4.0=py_0
+  - async_generator=1.10=py37h28b3542_0
```

Comment:
The idea is to provide users with simple recipes they can install, we don't need to provide them with fully resolved exported environments. See the metomi MO envs as a rough guide. Note that environment exports aren't portable as dependencies may differ on different platforms.

We only need to specify the primary dependencies here i.e. Python, Cylc, Rose, etc and don't need the `=py37h28b3542_0` bit.